components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -97,26 +119,57 @@ export default class SidebarCategory extends React.PureComponent<Props> {
         this.props.actions.setCategoryCollapsed(category.id, !isCollapsed);
     }
 
+    removeAnimation = () => {
+        if (this.newDropBoxRef.current) {
+            this.newDropBoxRef.current.classList.remove('animating');
+        }
+    }
+
     handleOpenDirectMessagesModal = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
         e.stopPropagation();
         this.props.handleOpenMoreDirectChannelsModal(e.nativeEvent);
     }
 
     render() {
         const {category, isCollapsed, channels} = this.props;
+        const {isNewCategory} = this.state;
+
         if (!category) {
             return null;
         }
 
-        if (category.type !== CategoryTypes.DIRECT_MESSAGES && (!channels || !channels.length)) {
+        if (!isNewCategory && category.type !== CategoryTypes.DIRECT_MESSAGES && (!channels || !channels.length)) {
             return null;
         }
 
         const renderedChannels = channels.map(this.renderChannel);
 
+        let newLabel;
+        let newDropBox;
         let directMessagesModalButton;
         let hideArrow = false;
-        if (category.type === CategoryTypes.DIRECT_MESSAGES) {
+        if (isNewCategory) {
+            newLabel = (
+                <div className='SidebarCategory_newLabel'>
+                    {localizeMessage('sidebar_left.sidebar_category.newLabel', 'new')}
```

Comment:
This can use a `FormattedMessage` component since we don't need the translated string as a plain string